tjpcov/main.py

```diff
@@ -793,6 +793,7 @@ def nmt_gaussian_cov(self, tracer_comb1=None, tracer_comb2=None,
                 cw = nmt_tools.get_covariance_workspace(f[1], f[2], f[3], f[4],
                                                         mn[1], mn[2], mn[3],
                                                         mn[4], self.outdir,
+                                                        lmax=int(ell[-1]),
```

Comment:
Why is setting `lmax` required for the workspace?

Reply:
@damonge will know the details. But looking at the source code, if lmax is None, it just sets whatever your field has (i.e. 3*nside).